example_test.go

```diff
@@ -0,0 +1,60 @@
+package parquet_test
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/segmentio/parquet-go"
+)
+
+func Example() {
+	// parquet-go uses the same struct-tag definition style as JSON and XML
+	type Contact struct {
+		Name string `parquet:"name"`
+		// "zstd" specifies the compression for this column
+		PhoneNumber string `parquet:"phoneNumber,optional,zstd"`
+	}
+
+	type AddressBook struct {
+		Owner             string    `parquet:"owner,zstd"`
+		OwnerPhoneNumbers []string  `parquet:"ownerPhoneNumbers,gzip"`
+		Contacts          []Contact `parquet:"contacts"`
+	}
+
+	f, _ := os.Create("testdata/data.parquet")
+	writer := parquet.NewWriter(f)
+	rows := []AddressBook{
+		{Owner: "UserA", Contacts: []Contact{
+			{Name: "Alice", PhoneNumber: "+15505551234"},
+			{Name: "Bob"},
+		}},
+		// Add more rows here.
+	}
+	for _, row := range rows {
+		if err := writer.Write(row); err != nil {
+			log.Fatal(err)
+		}
+	}
+	_ = writer.Close()
+	_ = f.Close()
```

Comment:
Is the `_ =` intended to silence linter rules?

Reply:
I wanted to indicate this function can return an error that you should check without adding 2 lines to the example to check it